docs/source/torchdata.datapipes.iter.rst

```diff
@@ -164,6 +164,7 @@ These DataPipes apply the a given function to each element in the DataPipe.
 
     BatchAsyncMapper
     BatchMapper
+    BatchThreadPoolMapper
```

Comment:
BTW, I plan to change the name of `BatchAsyncMapper` to `AsyncMapper`. Let's change `BatchThreadPoolMapper` to `ThreadPoolMapper`

Reply:
Will do

Reply:
The functional form will also be changed to `async_map`?

Reply:
Yeah